packages/dashboard/src/components/Wizard/WizardPage.tsx

```diff
@@ -1,21 +1,21 @@
 import * as React from 'react';
 import { ExampleCard, IComponentDemoPageProps, ComponentPage, PropertiesTableSet } from '@uifabric/example-app-base';
-import { SetupWizardBasicExample } from './examples/SetupWizard.Basic.Example';
 import { SetupWizardSubStepsExample } from './examples/SetupWizard.SubSteps.Example';
+import { PanelWizardExample } from './examples/PanelWizard.Example';
 
-const SetupWizardBasicExampleCode = require('!raw-loader!@uifabric/dashboard/src/components/Wizard/examples/SetupWizard.Basic.Example.tsx') as string;
 const SetupWizardSubStepsExampleCode = require('!raw-loader!@uifabric/dashboard/src/components/Wizard/examples/SetupWizard.SubSteps.Example.tsx') as string;
+const PanelWizardExampleCode = require('!raw-loader!@uifabric/dashboard/src/components/Wizard/examples/PanelWizard.Example.tsx') as string;
 
 export class WizardPage extends React.Component<IComponentDemoPageProps, {}> {
   public render(): JSX.Element {
     return (
       <ComponentPage
-        title="SetupWizard"
-        componentName="SetupWizard"
+        title="Wizard"
```

Comment:
Do we still have a basic example of setup wizard if this replaces it?

Reply:
It's not intend to replace it. For awhile I have three examples on this page. Later I felt that it's kind of redundant to have the basic example given that we already have the example with substeps, which showcase a super set of features.